docs/swagger/v1.json

```diff
@@ -1,710 +1,690 @@
 {
-    "openapi": "3.0.1",
-    "info": {
-      "title": "Azure SignalR Service REST API",
-      "version": "v1"
-    },
-    "paths": {
-      "/api/v1/health": {
-        "get": {
-          "description": "Get service health status",
-          "tags": [],
-          "responses": {
-            "200": {
-              "description": "Ok"
-            },
-            "503": {
-              "description": "ServiceUnavailable"
-            }
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Azure SignalR Service REST API",
+    "version": "v1"
+  },
+  "paths": {
+    "/api/v1/health": {
+      "get": {
+        "tags": [
+          "HealthApi"
+        ],
+        "summary": "Get service health status.",
+        "operationId": "HealthApi_GET_Index",
+        "responses": {
+          "200": {
+            "description": "The service is healthy"
+          },
+          "503": {
+            "description": "The service is temporarily unavailable"
           }
         }
       },
-      "/api/v1/hubs/{hub}": {
-        "post": {
-          "description": "Broadcast a message to all clients connected to target hub.",
-          "tags": [],
-          "parameters": [
-            {
-              "name": "hub",
-              "in": "path",
-              "required": true,
-              "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+      "head": {
+        "tags": [
+          "HealthApi"
+        ],
+        "summary": "Get service health status.",
+        "operationId": "HealthApi_HEAD_Index",
+        "responses": {
+          "200": {
+            "description": "The service is healthy"
+          },
+          "503": {
+            "description": "The service is temporarily unavailable"
+          }
+        }
+      }
+    },
+    "/api/v1/hubs/{hub}": {
+      "post": {
+        "tags": [
+          "HubProxyControllerV1"
```

Comment:
shall we hide this?

Reply:
any need to ?

Reply:
> Tagged operations may be handled differently by tools and libraries. For example, Swagger UI uses tags to group the displayed operations

Sounds useful

Reply:
shall we tag with some meaningful names instead of internal controller name, which sounds not useful to customer.

Reply:
updated